datastax-java-driver/src/main/scala/com/avast/sst/datastax/config/DatastaxDriverConfig.scala

```diff
@@ -0,0 +1,5 @@
+package com.avast.sst.datastax.config
+
+/** Configuration for datastax java driver.
+  */
+final case class DatastaxDriverConfig(basic: Basic, advanced: Advanced = Advanced.Default, profiles: List[Profile] = List.empty)
```

Comment:
`DatastaxJavaDriverConfig`

Reply:
Renamed to `CassandraDatastaxDriverConfig`